packages/@aws-cdk/aws-glue/README.md

```diff
@@ -23,6 +23,55 @@
 
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+## Connection
+
+A `Connection` allows Glue jobs, crawlers and development endpoints to access certain types of data stores.
+
+```ts
+new glue.Connection(stack, 'MyConnection', {
+  connectionType: glue.ConnectionTypes.NETWORK,
+  securityGroups: [securityGroup],
+  subnet,
+});
+```
+
+If you need to use a connection type that doesn't exist as a static member on `ConnectionType`, you can instantiate a `ConnectionType` object, e.g: `new glue.ConnectionType('NEW_TYPE')`.
+
+Different connection types require different connection properties which can be set via `ConnectionProps.properties`. 
+
+Subclasses of `Connection` are implemented to provide a connection type tailored props e.g. `MongoDBSourceConnection` which can be used as follows
+
+```ts
+new glue.MongoDBSourceConnection(stack, 'Connection', {
+    database: 'database',
+    collection: 'collection',
+});
+```
+
+if a specific connection type construct props does not expose all properties, additional properties can still be set as follows
```

Comment:
```suggestion
In case a specific connection type construct does not expose all properties, additional properties can still be set using the `properties` property.
```